services/agents/registry.go

```diff
@@ -1260,8 +1262,32 @@ func (r *Registry) StartPTSummaryAction(ctx context.Context, id, pmmAgentID stri
 	return nil
 }
 
-// StartPTMongoDBSummaryAction starts pt-pg-summary action on the pmm-agent.
-// The pt-mongodb-summary may require some of the following params: host, port, username, password.
+// StartPTPgSummaryAction starts pt-pg-summary action on the pmm-agent.
+// The function returns nil if ok, otherwise an error code
+func (r *Registry) StartPTPgSummaryAction(ctx context.Context, id, pmmAgentID, address string, port uint16, username, password string) error {
+	actionRequest := &agentpb.StartActionRequest{
```

Comment:
As far as I see PostgreSQL can be added using socket.
And pt-pg-summary cannot connect via socket. At least what docs say: https://www.percona.com/doc/percona-toolkit/3.0/pt-pg-summary.html 

So I think we need to return an error to the user here that we cannot get a summary for PostgreSQL via socket.
@JiriCtvrtka, and what is your opinion?

Reply:
@askomorokhov Yes, agree. Let me fix it.

Reply:
```

-h | --host=HOST |   | Host to connect to or local Unix socket for connection
-- | -- | -- | --

```

actually it can

Reply:
@BupycHuk Going to fix it back right now.